__tests__/lint-config/invalid-config--no-option/snapshot.js

```diff
@@ -0,0 +1,30 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`E2E lint-config test with option: { dirName: 'invalid-config--no-option', option: null } 1`] = `
+
+[1] .redocly.yaml:6:5 at #/lint/rules/context
+
+Property \`context\` is not expected here.
+
+4 | lint:
+5 |   rules:
+6 |     context: null
+  |     ^^^^^^^
+7 |
+
+Warning was generated by the spec rule.
+
+
+You have 1 warning.
+No configurations were defined in extends -- using built in recommended configuration by default.
+Warning! This default behavior is going to be deprecated soon.
+
+validating /Users/yevhensmoliy/redocly/projects/openapi-cli/__tests__/lint-config/__fixtures__/valid-openapi.yaml...
+/Users/yevhensmoliy/redocly/projects/openapi-cli/__tests__/lint-config/__fixtures__/valid-openapi.yaml: validated in <test>ms
```

Comment:
All the snapshots with this type of path will fail for the user locally. 
<img width="872" alt="image" src="https://user-images.githubusercontent.com/54616703/173570544-5cf91613-f379-43dd-918b-b575e82baee7.png">
Example:
<img width="866" alt="image" src="https://user-images.githubusercontent.com/54616703/173570619-1956ca2d-ecf6-40d4-882e-c01008705a59.png">

Reply:
Great catch, thx, will fix!